common/workunit/workunit.cpp

```diff
@@ -5058,6 +5058,118 @@ bool CWorkUnitFactory::restoreWorkUnit(const char *base, const char *wuid, bool
     return true;
 }
 
+bool CWorkUnitFactory::importWorkUnit(const char *wuid, const char *zapName, const char *xmlFile,
+    const char *importToIP, const char *importToPath, bool importQueryAssociatedFile)
```

Comment:
minor: importQueryAssociatedFile(s) or importAssociatedFiles

Reply:
Not sure that importWorkUnit() should be renamed to importAssociatedFiles(). In addition to importAssociatedFiles, the importWorkUnit() is also used to add the WU tree to dali.

Reply:
>minor: importQueryAssociatedFile(s) or importAssociatedFiles

I was referring to the parameter "bool importQueryAssociatedFile"

Reply:
Sorry, I got it now.